src/SigV4MessageHandler/Crypto.cs

```diff
@@ -0,0 +1,32 @@
+using System.Security.Cryptography;
+
+namespace InsightArchitectures.Utilities;
+
+#pragma warning disable SA1600
```

Comment:
enable it again

Reply:
Hmm, this is an internal class. Does it need documentation?

Reply:
No, i meant at the end of the file.